.github/workflows/backend-test-py3.6.yml

```diff
@@ -0,0 +1,64 @@
+name: backend [py3.6]
+on:
+  push:
+    branches:
+      - master
+      - releases/**
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-16.04
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        instance: [0, 1, 2]
+
+    env:
+      MIGRATIONS_TEST_MIGRATE: 1
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Check for python file changes
+        uses: getsentry/paths-filter@v2
+        id: changes
+        with:
+          token: ${{ github.token }}
+          filters: .github/file-filters.yml
+
+      - name: Set python version output
+        id: python-version
+        run: |
+          echo "::set-output name=python-version::$(awk 'FNR == 2' .python-version)"
+
+      # Until GH composite actions can use `uses`, we need to setup python here
+      - uses: actions/setup-python@v2
+        with:
```

Comment:
You forgot to add `if: steps.changes.outputs.backend == 'true'` here and the step above?

Is there no way to return from (not fail) the entire job in a step?

Reply:
Same thing applies to `.github/workflows/snuba-integration-test-py3.6.yml`.

Reply:
Yeah I don't think there's a way to exit early without failing